dialog/confirm.go

```diff
@@ -34,9 +42,9 @@ func NewConfirm(title, message string, callback func(bool), parent fyne.Window)
 			d.hideWithResponse(true)
 		},
 	}
-	d.create(container.NewGridWithColumns(2, d.dismiss, confirm))
+	d.create(container.NewHBox(layout.NewSpacer(), d.dismiss, confirm, layout.NewSpacer()))
```

Comment:
Is this change what we want? It was set up so the buttons are an even size before. I think that is how all the dialogs are, so isn't this inconsistent now?

If you want to change so they are not any more I don't think we need spacers as it is centred anyway.

Reply:
I made the change to be more consistent with the other dialogs. From what I could tell, the majority of dialogs (at least the custom ones and form) use a HBox with spacers.

However, I do agree that it probably makes more sense to use grids for even sizing. I can roll-back this change and open a follow-up PR to move the others to use a grid instead if you'd like?

Reply:
I would think this makes sense, yes

Reply:
Cool. I've just reverted the commit that introduced the change. Should be fine now